core/monad.py

```diff
@@ -400,7 +400,9 @@ def poll(cls, context):
     @property
     def instances(self):
         res = []
-        for ng in self.sv_trees:
+        sv_trees = [ng for ng in bpy.data.node_groups
+                    if ng.bl_idname in {'SverchCustomTreeType', 'SverchGroupTreeType'}]
+        for ng in sv_trees:
```

Comment:
why are you making this more complicated?

Reply:
This strings was migrated from `sv_trees` property of `SvNodeTreeCommon` class.

![2020-08-27_17-35-09](https://user-images.githubusercontent.com/28003269/91449122-cae00a80-e88b-11ea-86bd-9f78b4972482.png)

This is only place where this property was used. I don't think it worth to create separate method for this especially in base class. 

Even more I don't think that there should be such method in any NodeTree classes. More properly would be to write something like this:

```Python
bpy.data.node_groups.sv_trees = property([ng for ng in bpy.data.node_groups if ng.bl_idname in {'SverchCustomTreeType', 'SverchGroupTreeType'}])
```

Or atleast to create separate function in `utils` or where ever.

Reply:
or 
```python
# some node tree util
def get_nodetrees_in_blend(kind=["SverchCustomTreeType", "SverchGroupTreeType"]):
    return [ng for ng in bpy.data.node_groups if ng.bl_idname in set(kind)]


for ng in get_nodetrees_in_blend():
    ...
```
i'd prefer to have the logic of the nodetree kind filtering away from that area.

Reply:
The defaults values does not fit with function name. With such name `kinds` argument should not have any default values.

```Python
for ng in get_nodetrees_in_blend():
```
This code does not say about what is actually happening.

Reply:
you're right,  but seen as it would be part of a sverchok utility, it is fine to provide a default which covers all current sverchok groups by default.